### PR TITLE
kraken: Compute projected_centroid of at binarisation

### PR DESCRIPTION
**Pull request by kinnou02**
This greatly improve performances of autocompletion, we gain arround 50% in response time from kraken.
This is the first optimization of kraken autocompletion

### CODE REVIEW

File: source/georef/georef.h [hunk 1]
```diff
@@ -147,15 +147,17 @@ struct Way :public nt::Nameable, nt::Header{
     std::vector< HouseNumber > house_number_right;
     std::vector< std::pair<vertex_t, vertex_t> > edges;
     std::vector<nt::LineString> geoms;
+    nt::GeographicalCoord _projected_centroid;
```
Comment (pbougue) [inline source/georef/georef.h:150]: "kinda private"? ;-)

**Review by pbougue — COMMENTED** [review]
Easy win :)
Looks like there are no test on that currently... Should we put some?

**Comment by kinnou02** [comment]
There is some tests, and they will fail since I forgot to update them...

File: source/type/data.cpp [hunk 2]
```diff
@@ -61,7 +61,7 @@ namespace pt = boost::posix_time;
 
 namespace navitia { namespace type {
 
-const unsigned int Data::data_version = 68; //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 69; //< *INCREMENT* every time serialized data are modified
```
Comment (benoit-bst) [inline source/type/data.cpp:64]: :warning: actually I have also a change in my PR (https://github.com/CanalTP/navitia/pull/2597). 
But, if we merge the 2 PRs at the same time, we can keep `69` together

**Review by benoit-bst — APPROVED** [review]

**Comment by kinnou02** [comment]
#2600 provide better performance, I will do more bench later to see if this change is still needed.

**Pull request closed without merging**